### DISCUSSION TITLE
Why minZoom is not working while extent is defined on TileLayer?

### DISCUSSION THREAD

**Discussion by lianzhao** [visgl/deck.gl#7586] [Q&A] [answered]
I've noticed that with this `MVTLayer`
```js
    new MVTLayer({
      minZoom: 16,
      extent: myExtent,
      // ...
    });
```
The layer keeps loading tiles(zoom=16) while the actual zoom is less that 16. Than I found the [document](https://deck.gl/docs/api-reference/geo-layers/tile-layer#minzoom) said `When underzoomed (i.e. zoom < minZoom), the layer will not display any tiles unless extent is defined, to avoid issuing too many tile requests.`

May I ask why extent matters?

**Comment by Pessimistress** [comment, marked as answer]
`minZoom` indicates the availability of the tile service. The layer will **attempt** to render whenever reasonable. If tiles at the current zoom level are not available, then the layer will have to load tiles at the closest available z. That can mean a lot of tiles if you are underzoomed, potentially overwhelming your network connection and memory. To prevent that, you are required to supply an `extent` in order to have the layer visible at < minZoom.

If you are looking for a way to hide the layer at certain zoom levels, `minZoom` does not do it for you. You should set the `visible` prop to `false`.

**Reply by lianzhao** [reply]
Yes `visible` prop is what I was looking for, thank you for your help!
